### DISCUSSION TITLE
About the Special Variables used for Output files

### DISCUSSION THREAD

**Discussion by hsdk0000** [mifi/lossless-cut#1796] [Q&A]
Hello,I've been using your software for a while and have been loving it! But I'm not used to github so I don't know where to look for a full list of what I can do with the output names ( for example : [name]${EXT}), I just want to look for a documentation of some sorts about those variables regarding the file's name, thank you very much again for the great software !

**Comment by andrewdbond** [comment]
Within the Export screen, if you click the filename, the section expands listing possible variables, and with a help link that links to https://github.com/mifi/lossless-cut/blob/master/import-export.md#customising-exported-file-names

**Reply by hsdk0000** [reply]
Thank you very much, sorry for the late replies !

**Reply by andrewdbond** [reply]
Thanks. If satisfied, @hsdk0000 as this discussion's 'Author' it appears you can mark the comment as the answer so that this discussion is no longer tagged as "Unanswered". https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#marking-a-comment-as-an-answer
